doc/devel/documenting_mpl.rst

```diff
@@ -16,18 +16,19 @@ are needed to build the documentation. They are listed in the file
 `doc-requirements.txt <https://github.com/matplotlib/matplotlib/blob/master/doc-requirements.txt>`_
 as well as listed below:
 
-1. Sphinx-1.3 or later(Version 1.5.0 is not supported)
-2. numpydoc 0.4 or later
-3. IPython
-4. Mock
-5. colorspacious
-6. pillow
-7. graphviz
+* Sphinx>=1.3, !=1.5.0, !=1.6.4
+* colorspacious
+* IPython
+* mock
+* numpydoc>=0.4
+* Pillow
+* sphinx-gallery>=0.1.12
+* graphviz
 
 .. note::
 
   * You'll need a minimal working LaTeX distribution for many examples to run.
-  * `Graphviz <http://www.graphviz.org/Download.php>`_ is not a python package, and needs
+  * `Graphviz <http://www.graphviz.org/Download.php>`_ is not a python package, and needs 
```

Comment:
Unnecessary change?

Reply:
fixed, dunno how the spare space got there